nautobot/ipam/views.py

```diff
@@ -63,39 +65,99 @@
 #
 
 
-class NamespaceUIViewSet(NautobotUIViewSet):  # 3.0 TODO: Remove ObjectBulkCreateViewMixin, no longer used
+class NamespaceUIViewSet(NautobotUIViewSet):
     form_class = forms.NamespaceForm
     bulk_update_form_class = forms.NamespaceBulkEditForm
     filterset_class = filters.NamespaceFilterSet
     filterset_form_class = forms.NamespaceFilterForm
     queryset = Namespace.objects.all()
     serializer_class = serializers.NamespaceSerializer
     table_class = tables.NamespaceTable
-
     object_detail_content = object_detail.ObjectDetailContent(
-        panels=(
-            object_detail.ObjectFieldsPanel(section=SectionChoices.LEFT_HALF, weight=100, fields="__all__"),
-            object_detail.ObjectsTablePanel(
-                table_class=tables.VRFTable,
-                section=SectionChoices.FULL_WIDTH,
-                weight=100,
-                table_filter="namespace",
+        panels=(object_detail.ObjectFieldsPanel(section=SectionChoices.LEFT_HALF, weight=100, fields="__all__"),),
+        extra_tabs=(
+            object_detail.DistinctViewTab(
+                weight=800,
+                tab_id="vrfs",
+                label="VRFs",
+                url_name="ipam:namespace_vrfs",
             ),
-            object_detail.ObjectsTablePanel(
-                table_class=tables.PrefixTable,
-                section=SectionChoices.FULL_WIDTH,
-                weight=100,
-                table_filter="namespace",
+            object_detail.DistinctViewTab(
+                weight=800,
+                tab_id="prefixes",
+                label="prefixes",
+                url_name="ipam:namespace_prefixes",
             ),
-            object_detail.ObjectsTablePanel(
-                table_class=tables.IPAddressTable,
-                section=SectionChoices.FULL_WIDTH,
-                weight=100,
-                table_filter="parent__namespace",
+            object_detail.DistinctViewTab(
+                weight=800,
+                tab_id="ipaddresses",
+                label="ipaddresses",
+                url_name="ipam:namespace_ipaddresses",
             ),
-        )
+        ),
     )
 
+    @action(detail=True, url_path="vrfs")
+    def vrfs(self, request, *args, **kwargs):
```

Comment:
Ideally we'd have a pattern for each of these action views to also have a uniquely defined `object_detail_content` and so be able to use things like `ObjectsTablePanel` - that may require some upfront dev work in core to make it possible though.